setup.py

```diff
@@ -34,6 +34,7 @@
         'django-timezone-field>=2.0rc1',
         'django-localflavor>=1.1',
         'swapper>=0.3.0',
+        'django-recaptcha',
```

Comment:
need also the `axes` as Waseem mentioned.
I also suggest to add version lock for the minimum`django-recaptcha` required